src/scripts/docs/FeedbackApp.jsx

```diff
@@ -126,7 +127,7 @@ class FeedbackApp extends React.Component {
           </p>
           <div className="site-example-toasts u-m-b-large">
             <Toast type="default" body="Default toast notification" />
-            <Toast type="success" body="Success toast notification" />
+            <Toast type="success" body="Success toast notification" subText="Please allow sometime before this activity is displayed" />
```

Comment:
Is there a reason we want to default the `subText` to this for all success toasts? To me this text seems very specific to something like RhinoBlast that is an asynchronous process. Would it better to default this to an empty subtext?

Reply:
This is an example file where we show how subText working.